### ISSUE TITLE
Python 3.10

### ISSUE THREAD

**Issue by AhmadNaruto** [termux/termux-packages#7644]
### Package description

New release candidate with lot of cool feature 

### Home page URL

https://www.python.org/

### Source code URL

https://github.com/python/cpython/releases/tag/v3.10.0b2

### Packaging policy acknowledgement

- [X] I have read and understand the [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).

**Comment by Grimler91** [comment]
It's a *release candidate*. Replacing our (stable) python 3.9 package with a release candidate seems like a bad idea.

**Comment by AhmadNaruto** [comment]
i know it. but lets say its place in unstable repo termux

**Comment by Grimler91** [comment]
Right, re-opening then in case someone wants to package it along side python3.9

**Comment by xtkoba** [comment]
Note that Python 3.10.0 (final) was released yesterday (in UTC):

https://github.com/python/cpython/releases/tag/v3.10.0
